x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestsDataLoader.java

```diff
@@ -137,17 +138,33 @@ public static void main(String[] args) throws IOException {
         }
 
         try (RestClient client = builder.build()) {
-            loadDataSetIntoEs(client);
```

Comment:
This change comes as a consequence of `main` method not working anymore because of `Exception in thread "main" java.security.AccessControlException: access denied ("org.elasticsearch.secure_sm.ThreadPermission" "modifyArbitraryThreadGroup")`. The `main` doesn't work being executed from within IDE (run from gradle or as a Java class) or outside of it with gradle. The exception comes from randomization code inside Lucene

```
	at org.apache.lucene.tests.util.LuceneTestCase.random(LuceneTestCase.java:800)
	at org.elasticsearch.test.ESTestCase.randomFrom(ESTestCase.java:1003)
	at org.elasticsearch.test.rest.ESRestTestCase.randomSupportedContentType(ESRestTestCase.java:2409)
	at org.elasticsearch.test.rest.ESRestTestCase.addXContentBody(ESRestTestCase.java:2413)
	at org.elasticsearch.test.rest.ESRestTestCase.newXContentRequest(ESRestTestCase.java:2426)
	at org.elasticsearch.test.rest.ESRestTestCase.createIndex(ESRestTestCase.java:1709)
```